.travis.yml

```diff
@@ -0,0 +1,2 @@
+language: minimal
+script: ./run_tests.sh
```

Comment:
Run the tests from the shell script run_tests.sh

Reply:
source: https://github.com/dsaltares/godot-ci-example